README.md

```diff
@@ -104,9 +104,42 @@ information [inside the Change Data Capture section](#change-data-capture)).
 | `columns`        | list of column names that should be included in each Record's payload, by default includes all columns | false    | `id,name,age`                               |
 | `batchSize`      | size of rows batch. Min is 1 and max is 100000                                                         | false    | `100`                                       |
 
+## Destination
+
+The Oracle Destination takes a `record.Record` and parses it into a valid SQL query. The Destination is designed to
+handle different payloads and keys. Because of this, each record is individually parses and upsertes.
+
+### Table name
+
+If a record contains a `table` property in its metadata it will be inserted in that table, otherwise it will fall back
+to use the table configured in the connector. Thus, a Destination can support multiple tables in a single connector, as
+long as the user has proper access to those tables.
+
+### Upsert Behavior
+
+If the target table already contains a record with the same key, the Destination will upsert with its current received
+values. Because Keys must be unique, this can lead to overwriting and potential data loss, so the keys must be correctly
```

Comment:
```suggestion
If the target table already contains a record with the same key, the Destination will still upsert the new record
value. since keys must be unique, this can lead to overwriting and potential data loss, so the keys must be correctly
```

Reply:
Updated. 
`since keys must be unique` - is a new sentence, right?

Reply:
yes, so it's better to have `Since` capitalized.